### DISCUSSION TITLE
reproduce results of T2I-Compbench

### DISCUSSION THREAD

**Discussion by Gus-Guo** [PixArt-alpha/PixArt-alpha#146] [Q&A]
Hi, could you share the detail about the evaluation of T2I-Compbench? I use Pixart-alpha-512x512 ckpt and only got results of 0.4 on color attribute, with a cfg scale of 4. I am not sure whether there is something wrong...

**Comment by Gus-Guo** [comment]
> Hi, could you share the detail about the evaluation of T2I-Compbench? I use Pixart-alpha-512x512 ckpt and only got results of 0.4 on color attribute, with a cfg scale of 4. I am not sure whether there is something wrong...

got similar results when I use diffusers to inference Pixart-alpha-512x512, with the default setting

**Comment by WeiLi-THU** [comment]
I got a similar problem when I used diffuser to reproduce the SDXL score with the default setting, but when I set the cfg to 10.0, the results even exceeded the figure reported in PixelArt-alpha paper, 64.39 vs 63.69 specifically. So, could you share the details of the inference code of the T2I models reported in the paper? It will be a big help to the followers to make a fair comparison.
